lib/tasks/email_and_unsubscribe_archived_topics.rake

```diff
@@ -1,12 +1,16 @@
 namespace :archived_topics do
   desc "Send a bulk email to subscribers to these archived Specialist Topics."
-  task :send_emails, [] => :environment do |_t, _args|
-    topic_urls.each do |topic|
+  task :send_emails, [:check_output] => :environment do |_t, args|
+    args.with_defaults(check_output: "false")
```

Comment:
Is it safer to have `check_output: "true"` by default. And then you have to pass in an extra argument to make the task run for real?

Instead of lines 32 to 41 you could just wrap the `BulkSubscriberListEmailBuilder` in an unless `check_output: false`. So you see exactly what's going to happen?

Reply:
Good shout - thanks

Reply:
Addressed in https://github.com/alphagov/email-alert-api/commit/3585b4808f5fce5e4b321d7b2633490aea90a5ee